cypress.env.example.json

```
@@ -0,0 +1,6 @@
+{
+    "STAGE_HOST":"",
```

```suggestion
    "QA_HOST":"",
```

you mentioned QA_HOST in scripts update the env variables